datalad/metadata/definitions.py

```diff
@@ -0,0 +1,93 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Metadata (key) definitions"""
+
+
+# for maximum compatibility with git-annex' metadata setup, _keys_ in this
+# dictionary should be all lower-case, and be limited to alphanumerics, plus
+# '_', '-', and '.' -- except for JSON-LD keywords (which start with '@' and
+# will be ignored in the context of git-annex metadata
+common_key_defs = {
+    "@vocab": {
+        'def': "http://schema.org/",
+        'descr': 'base vocabulary'},
+    "doap": {
+        'def': "http://usefulinc.com/ns/doap#",
+        'descr': 'ontology for the description of a project'},
+    "author": {
+        'def': "schema:author",
+        'descr': 'author of some resource'},
```

Comment:
We talked about this before, I believe that hand-written metadata will be rare, but I don't object examples -- give them to me and I'll put them in.